.github/workflows/test-install-script.yml

```diff
@@ -207,21 +213,43 @@ jobs:
       run: |
         bash utils/uninstall.sh -q -V
     - name: Install WasmEdge specific release (0.10.0) with all extensions
+      if: ${{ matrix.name != 'MacOS-arm64' }}
       run: |
         bash utils/install.sh -e all -v 0.10.0 -D
     - name: Uninstall WasmEdge with all extensions
+      if: ${{ matrix.name != 'MacOS-arm64' }}
       run: |
         bash utils/uninstall.sh -q -V
-    - name: Plugins Install check
+    - name: Plugins Install check MacOS-x86
+      if: ${{ matrix.name != 'MacOS-arm64' }}
       run: |
         bash utils/install.sh -v 0.10.0-alpha.1
         ls ~/.wasmedge/plugin/ | grep .dylib && echo "Pass: Plugins found" || (echo "Fail: Plugins not found" && exit 1)
+        bash utils/install.sh -v 0.13.1 --plugins wasmedge_tensorflow
+        ls ~/.wasmedge/plugin/ | grep libwasmedgePluginWasmEdgeTensorflow.dylib && echo "Pass: Plugins found" || (echo "Fail: Plugins not found" && exit 1)
+        bash utils/install.sh -v 0.13.1 --plugins wasmedge_tensorflowlite
+        ls ~/.wasmedge/plugin/ | grep libwasmedgePluginWasmEdgeTensorflowLite.dylib && echo "Pass: Plugins found" || (echo "Fail: Plugins not found" && exit 1)
         bash utils/install.sh -v 0.10.0-alpha.1 -p ~/new_wasmedge
         ls ~/new_wasmedge/plugin/ | grep .dylib && echo "Pass: Plugins found" || (echo "Fail: Plugins not found" && exit 1)
         # MacOS container does not have access for /usr directory
         # bash utils/install.sh -v 0.10.0-alpha.1 -p /usr
         # ls /usr/lib/wasmedge/ | grep libwasmedgePluginWasmEdgeProcess.dylib && echo "Pass: Plugins found" || (echo "Fail: Plugins not found" && exit 1)
+    
+    # macos-arm64 does not have plugins
```

Comment:
We do have plugins for macOS arm64. @q82419 Could you please identify them?

Reply:
The default plugins that come with x86 are not there ig

Reply:
![CleanShot 2023-07-11 at 22 26 45](https://github.com/WasmEdge/WasmEdge/assets/2776756/d7443a0d-a11c-41ab-8e70-5c75a0d415c3)
![CleanShot 2023-07-11 at 22 26 59](https://github.com/WasmEdge/WasmEdge/assets/2776756/290a094f-ebca-4a92-8ff2-7c3d79620547)
![CleanShot 2023-07-11 at 22 27 10](https://github.com/WasmEdge/WasmEdge/assets/2776756/7623331f-1d4f-4f28-bd75-1beac7645d87)
![CleanShot 2023-07-11 at 22 27 34](https://github.com/WasmEdge/WasmEdge/assets/2776756/a2bfb69e-9c96-4c19-bb8e-a22994d7027b)
![CleanShot 2023-07-11 at 22 27 50](https://github.com/WasmEdge/WasmEdge/assets/2776756/16a3dda7-043a-41fe-8a4a-2973a30cd9ac)

Reply:
The default plugins that come in the wasmedge installation package and don't need external installation.

Reply:
bash utils/install.sh -v 0.13.1
ls ~/.wasmedge/plugin/ | grep .dylib && echo "Pass: Plugins found" || (echo "Fail: Plugins not found" && exit 1)

This fails ig

Reply:
Also we have aarch64 for 0.13.0 and arm64 for 0.13.1, can you please fix that?

Reply:
The darwin_aarch64 are all removed from the 0.13.0 release assets.

Reply:
Yes, for some https://github.com/WasmEdge/WasmEdge/releases/download/0.13.0-alpha.1/WasmEdge-0.13.0-alpha.1-darwin_aarch64.tar.gz it was there

Reply:
> bash utils/install.sh -v 0.13.1
> ls ~/.wasmedge/plugin/ | grep .dylib && echo "Pass: Plugins found" || (echo "Fail: Plugins not found" && exit 1)

Can you check this for macos-arm64?

Reply:
The error occurs on the `0.13.0-alpha.1`. Fixed that.

Reply:
`0.13.1` does not seem to have those https://github.com/WasmEdge/WasmEdge/actions/runs/5530355689/jobs/10089658182?pr=2608#step:14:6283

Reply:
Thanks